msg_hash.h

```diff
@@ -3269,6 +3269,12 @@ unsigned *msg_hash_get_uint(enum msg_hash_action type);
 
 void msg_hash_set_uint(enum msg_hash_action type, unsigned val);
 
```

Comment:
Could you please add a comment explaining these functions? Future devs need to know what a `wideglyph_str` is without referring to the source code